components/Cards/index.js

```diff
@@ -17,3 +17,76 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+//axios.get
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+//make sure to keep loking at the response given in console!
+    .then(res => {
+        const { bootstrap, javascript, jquery, node, technology } = res.data.articles
+
+        // console.log(bootstrap)
+        //notice the data you get back
+        //map over so that you can retrive the data and iterate over it and display it onn\ the page
+        bootstrap.map(card => {
+            const cardDiv = document.querySelector('.cards-container')
+            cardDiv.appendChild(cardCreator(card))
+        })
```

Comment:
Instead of mapping over each array one at a time, you could shorten this by turning the object into an array using Object.values(). If this site ever added a new topic you would have to add that topic in here and map over that new array. Here is how I solved it: 

axios
  .get("https://lambda-times-backend.herokuapp.com/articles")
  .then(res => {
    const newArray = Object.values(res.data.articles);
    newArray.map(articles => {
      articles.map(article => {
        const container = document.querySelector(".cards-container");
        container.appendChild(CardComponent(article));
      });
    });
  })
  .catch(err => {
    console.log(err);
  });